README.md

```diff
@@ -15,10 +15,17 @@ Currently, you can use *Hummingbird* to convert your trained traditional ML mode
 ## Installation
 
 Hummingbird was tested on Python >= 3.5 on Linux, Windows and MacOS machines.  It is recommended to use a virtual environment (See: [python3 venv doc](https://docs.python.org/3/tutorial/venv.html) or [Using Python environments in VS Code](https://code.visualstudio.com/docs/python/environments).)
+
+
+Install the Hummingbird package:
 ```
-pip install hummingbird-ml
+pip install hummingbird-ml[extra]
 ```
 
+This includes optional dependencies lightgbm and xgboost. If you do not need those, you can use:
```

Comment:
Should we do the opposite? First having `pip install hummingbird-ml` (I think this should be the default) and then say, if you need all the extra dependencies such as Sklearn, LightGBM, etc, you can use: `pip install hummingbird-ml[extra]`.

Reply:
Now that we have SKL as a req dep, yes, good point!